go.mod

```diff
@@ -592,7 +592,7 @@ require github.com/lorenzosaino/go-sysctl v0.3.1
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
-	github.com/DataDog/agent-payload/v5 v5.0.100
+	github.com/DataDog/agent-payload/v5 v5.0.103-0.20240102141342-488df63c5468
```

Comment:
It needs to be updated when https://github.com/DataDog/agent-payload/pull/281 is merged